demo/screens/BuyLandScreen.js

```diff
@@ -4,19 +4,18 @@ import { removeLandForSale } from "../actions";
 import BuyLand from "@presentational/BuyLand";
 import PropTypes from "prop-types";
 
-import ContractsABIs from "@constants/ContractsABIs";
-const { estateABI, marketplaceABI } = ContractsABIs;
 import ContractsAddresses from "@constants/ContractsAddresses";
-const { estateAddress, marketplaceAddress } = ContractsAddresses;
+const { ESTATE_ADDRESS, MARKETPLACE_ADDRESS } = ContractsAddresses;
 
 import { Action } from "tasit-sdk";
-const { Contract } = Action;
+const { ERC721, Marketplace } = Action;
+const { Estate } = ERC721;
+const { Decentraland } = Marketplace;
 import { approveManaSpending, manaFaucetTo } from "./helpers";
 
 export class BuyLandScreen extends React.Component {
-  // TODO: Switch to new DecentralandEstate() once the SDK includes that
-  estateContract = new Contract(estateAddress, estateABI);
-  marketplaceContract = new Contract(marketplaceAddress, marketplaceABI);
+  estateContract = new Estate(ESTATE_ADDRESS);
+  marketplaceContract = new Decentraland(MARKETPLACE_ADDRESS);
```

Comment:
Hmmm. Now that I'm seeing this it feels awkward that it's called Decentraland. Maybe as a short-term change we can import it `as` another name?

Reply:
Yes,
`const { Decentraland: Marketplace } = Marketplace` sounds good?

Reply:
Works for me!

Reply:
Done.
I've used `const { Decentraland: DecentralandMarketplace } = Marketplace`
because `Marketplace` var is already defined.
Or you prefer?
`const { Decentraland: Marketplace } = Action.Marketplace`

Reply:
The option you went with works for me. Tough choice, haha. Resolving this.